package.json

```diff
@@ -63,7 +63,7 @@
     "@vue/test-utils": "^1.1.2",
     "babel-core": "^7.0.0-bridge.0",
     "cache-loader": "^4.1.0",
-    "electron": "^10.3.0",
+    "electron": "^12.0.9",
```

Comment:
Please also update `@types/node` in `devDependencies` (it looks like it wants `^14.6.2` now).

Reply:
Sure. I was trying to do everything with node 14 (and actually replaced my brew node with nvm node so I have better control of the versions). Does this mean all devs have to move to node 15?